Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/TMP_KeyboardInputField.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    [RequireComponent(typeof(TMP_InputField))]
+    public class TMP_KeyboardInputField : KeyboardInputFieldBase<TMP_InputField>
+    {
+#if WINDOWS_UWP
+    protected override void ClearText() => inputField.text = string.Empty;
```

Comment:
nit: tab over to match file

Reply:
Ok now I have to ask, what does "nit" mean? Is there a shortcut section somewhere where I can read about all these?

Reply:
it's short I guess for nit-picking. Generally nit-picking has a negative connotation as someone who is consequently pointing out small flaws that aren't that meaningful.

In CRs, it's used as a, "Hey I know this is annoying and minor, but we should fix this...yet also don't block the PR on this one small thing."